src/main/java/org/ambraproject/wombat/config/YamlConfiguration.java

```diff
@@ -98,14 +99,25 @@ public String getRootPagePath() {
     return input.rootPagePath;
   }
 
-  @Override
-  public ThemeTree getThemes(Collection<? extends Theme> internalThemes, Theme rootTheme) throws ThemeTree.ThemeConfigurationException {
-    return ThemeTree.parse(input.themes, internalThemes, rootTheme);
+  private static ThemeSource<?> parseThemeSource(Map<String, ?> map) {
+    String type = (String) map.get("type");
+    if (type == null) throw new RuntimeException("Theme source must have type");
+    // Type is a hook for other ThemeSource types that may exist in the future.
+    // Currently the only supported value is "filesystem".
+    if (type.equals("filesystem")) {
```

Comment:
The main anticipated use case is some sort of remote source or service, something we access by URL.

An enum sounds fine; will add.